### PR TITLE
Feature/post parking lot route

### PR DESCRIPTION
**Pull request by samuelng678**
# Administrative Info
Make sure your branch name conforms to: `feature/hotfix/.../describe_your_branch_with_underscores

### Changes
What changes did you make?
Implemented POST function in parking_lot.js
## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tested my program thoroughly
- [ ] Did you add/sub fields in the environment file? if so please update .env_sample

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.
Screen Shot 2021-07-06 at 13 08 02

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |

### CODE REVIEW

**Review by anguyen698 — CHANGES_REQUESTED** [review]
Change Lots to lot_model

**Review by Eldar827 — APPROVED** [review]
lgtm

**Review by YuyusanK — CHANGES_REQUESTED** [review]
Looks good, but I'm sure if you pulled from master before making the PR. Please pull from the master before making the PR.

File: routes/parking_lot.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const Lots = require("../database/model/lotModel");
 
 /* POST parking_lot route */
 router.post("/parking_lot", (req, res) => {
```
Comment (YuyusanK) [inline routes/parking_lot.js:7]: This is an async function. Please add `async function` before `(req, res)`. Please remove `=>`

File: routes/parking_lot.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ const Lots = require("../database/model/lotModel");
...
+      lot_model.insert(req.body);
```
Comment (YuyusanK) [inline routes/parking_lot.js:8]: Please add `await` before `lot_model`

**Review by YuyusanK — CHANGES_REQUESTED** [review]

**Pull request closed without merging**